Library/Homebrew/missing_formula.rb

```diff
@@ -26,6 +26,12 @@ def blacklisted_reason(name)
           Minimal installation:
             brew cask install basictex
         EOS
+        when "asymptote" then <<~EOS
+          Asymptote is bundled with MacTeX / TeX Live. When that is installed, Asymptote
```

Comment:
I'd still like to see a `brew cask install` command here somewhere, thanks; either replacing or in addition to the `tlmgr` below.

Reply:
My feeling is that people can google for MacTeX (and then install upstream [_MacTeX.pkg_](https://tug.org/mactex/mactex-download.html), which is probably what most TeX users on macOS do) or do `brew install latex` (or now `brew install mactex`) and find out about the casks that way, via the `when "tex",…` block on the preceding lines.

That `tlmgr` line shows how to install Asymptote, which is what the user has requested. But please feel free to add something additional about `brew cask install` when merging the PR if you think it will add clarity.

Reply:
I'd like this to be consistent with the other lines most of which recommend a `brew install` or `brew cask install`, thanks. This is pretty wordy compared to the others to the extent that I'm not convinced it is actually making things easier in its current version.

Reply:
The point of this hint is to tell Brew users looking for Asymptote that the way to install it is not via `brew install`, as per Homebrew/legacy-homebrew#23029. The initial PR was less wordy.

The Homebrew maintainers should feel free to wordsmith the hint to their satisfaction or just close the PR. Thanks.

Reply:
@jmarshall I literally don't understand how to install Asymptote given these instructions. If I don't understand given the context and history of this PR I'd be surprised if this is going to be a useful message to others. I would update the instructions myself but, again, I don't understand what they need to be changed to.

Reply:
Okay, that is new information. [Asymptote is closely associated with LaTeX](http://asymptote.sourceforge.net) so users looking to install it will likely be familiar with TeX and have a better background for reading those instructions. Perhaps it would be useful for a Homebrew maintainer who is familiar with TeX to also review this PR.

In any case: Updated the instructions to be less wordy and to clarify that it is recommended to just install `MacTeX` and that will probably give you Asymptote without further fuss.